### PR TITLE
SharedCode as an Android Library

### PR DESCRIPTION
**Pull request by mike-mathieu**
Just a pretty lean cut from the https://github.com/touchlab/DroidconKotlin example gradle to get it to work for us.

### CODE REVIEW

File: Android/.idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default (1)" />
```
Comment (mike-mathieu) [inline Android/.idea/codeStyles/codeStyleConfig.xml]: Any idea how to get rid of this? Keeps auto-populating when I try to commit

File: Android/SharedCode/build.gradle [hunk 2]
```diff
@@ -1,8 +1,26 @@
 apply plugin: 'kotlin-multiplatform'
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 29
+
+    defaultConfig {
+        minSdkVersion 23
```
Comment (DavidCorrado) [inline Android/SharedCode/build.gradle]: Probably should find a way to share this version number between libraries.  With all of these.

File: Android/.idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (DavidCorrado) [reply Android/.idea/codeStyles/codeStyleConfig.xml]: Can you reset to the default code style settings?

**Review by DavidCorrado — CHANGES_REQUESTED** [review]

File: Android/.idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (mike-mathieu) [reply Android/.idea/codeStyles/codeStyleConfig.xml]: if I apply Default it changes to:
`<option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />`
and removes the `USE_PER_PROJECT_SETTINGS` line

Comment (DavidCorrado) [reply Android/.idea/codeStyles/codeStyleConfig.xml]: I would say thats probably fine and my settings need to be reset too =)

**Review by DavidCorrado — APPROVED** [review]

**Pull request merged**